Update oracle.mdx missing feature flags (#23517)
hashicorp/vault: website/content/api-docs/secret/databases/oracle.mdx

  and password fields. See the [databases secrets engine docs](/vault/docs/secrets/databases#disable-character-escaping)
  for more information. Defaults to `false`.

> - `split_statements` `(boolean: true)` - Indicates the statements specified should be split on semi-colons.
>   When true, this allows using a single string to specify multiple SQL statements. If set to false this allows for more 
>   complex queries such as stored or inline procedures.
> 
> - `disconnect_sessions` `(boolean: true)` -  Indicates the plugin should disconnect any open sessions prior to running the 
>   revocation statements. If set to false, the revocation statements should include session disconnect logic.

Reviewer: ```suggestion
- `split_statements` `(boolean: true)` - Indicates that the provided statements
  specified should be split on semi-colons. Set `split_statements` to true to
  specify multiple SQL statements in a single string. Set `split_statements` to
  `false` for complex queries such as stored or inline procedures.

- `disconnect_sessions` `(boolean: true)` -  Indicates the plugin should
  disconnect any open sessions before running revocation statements. If
  `disconnect_sessions` is `false`, the revocation statements **must** include
  session disconnect logic.
```

Style correction: write in active voice with declarative statements
